bot/cogs/help.py

```diff
@@ -0,0 +1,234 @@
+from inspect import Parameter
+import textwrap
+import typing as t
+
+import discord
+from discord.ext import commands, menus
+import discord.utils
+
+
+class HelpSource(menus.ListPageSource):
+    """The Help menu."""
+
+    def __init__(
+            self,
+            signature: t.Callable,
+            filter_commands: t.Coroutine,
+            prefix: str,
+            author: discord.User,
+            cogs: dict
+    ) -> None:
```

Comment:
This is still not the correct way to do it, as I mentioned (now for the third time) we need to indent the arguments by 4 spaces (1 indent size) not by 8 as you're doing. There is no reason to use double indent here
**This also applies for all functions with over-indented parameters**
```suggestion
    def __init__(
        self,
        signature: t.Callable,
        filter_commands: t.Coroutine,
        prefix: str,
        author: discord.User,
        cogs: dict
    ) -> None:
```

Reply:
As I already told you : C0330 : wrong hanging indentation before block
The function def must be indented more than the body, to avoid confusion

Reply:
`C0330` is also an error that isn't accepted by `flake8` on default and it isn't an issue here as the parameters are visually separated because of the ending bracket position being on the same level as the function definition. That poses a visual separation for where the function ends which is quite easily noticeable and there is no need for the double indent on those variables